### PR TITLE
fix(oauth-providers): oauth-middleware: load `env.SECRETS` with hono/adapter

### PR DESCRIPTION
**Pull request by jokester**
Hi, when using the middleware in node.js, I found it does not automatically read from process.env . Use of `hono/adapter` should fix this.

(correction welcome, I'm rather new to honojs)

### CODE REVIEW

**Comment by yusukebe** [comment]
Hi @jokester 

Thanks for the PR. Could you run `yarn chageset` at the top of the project and push the changeset created? This will be a path change.

**Comment by jokester** [comment]
@yusukebe I think I did it . Please check 🙏🏽

File: .changeset/dull-frogs-glow.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@hono/oauth-providers': minor
```
Comment (yusukebe) [inline .changeset/dull-frogs-glow.md:2]: Sorry! I could not tell you correctly. It's a patch change. So rewrite `minor` to `patch`.

Comment (jokester) [reply .changeset/dull-frogs-glow.md]: @yusukebe Fixed again 👍🏽

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@jokester 

Thank you so much! I'll merge this now.

**Pull request merged**